src/System.Memory/tests/Memory/OwnedMemory.cs

```diff
@@ -20,6 +20,27 @@ public static void MemoryFromOwnedMemoryInt()
             OwnedMemory<int> owner = new CustomMemoryForTest<int>(a);
             Memory<int> memory = owner.Memory;
             memory.Validate(91, 92, -93, 94);
+            memory.Slice(0, 4).Validate(91, 92, -93, 94);
+            memory.Slice(1, 0).Validate();
+            memory.Slice(1, 1).Validate(92);
+            memory.Slice(1, 2).Validate(92, -93);
```

Comment:
I don't think these tests should pass given the current implementation of owner.Memory and memory.Slice.

Reply:
They do.